### DISCUSSION TITLE
Web Link Down

### DISCUSSION THREAD

**Discussion by may-96** [cdmoro/bootstrap-vue-3#869] [General]
Unable to access the website for documentation, Getting 404!

![image](https://user-images.githubusercontent.com/26191959/211271154-fae013cb-c473-4ea7-8308-76ad84852696.png)

**Comment by VividLemon** [comment]
Duplicate of https://github.com/bootstrap-vue/bootstrap-vue-next/issues/866 . Tldr, I know.

**Comment by VividLemon** [comment]
https://bootstrap-vue.github.io/bootstrap-vue-next/
